Src/PerformanceCollector/Perf.Shared.NetStandard20Net45/StandardPerformanceCollector/StandardPerformanceCollector.cs

```diff
@@ -94,6 +94,7 @@ public void RefreshCounters()
                 perfCounterName,
                 this.win32Instances,
                 this.clrInstances,
+                true,
```

Comment:
nit: consider using parameter names here. helps the reader understand what the "true" value is set to.